### DISCUSSION TITLE
Which of the four .pb models should be used in formal DPMD simulation？

### DISCUSSION THREAD

**Discussion by h840473807** [deepmodeling/deepmd-kit#2630] [Q&A] [answered]
In the training of Dp model, dpgen generates four .pb files, 00: "tanh, tanh", 01: "tanh, gelu", 02: "gelu, tanh" and 03: "gelu, gelu". The rmse_f_trn of these four models is different. In the training of my system, the rmse_f_trn of 00 is maximum while that of 02 is minimum. Does this mean that 02 model should be used to generate trajectory in formal DPMD simulation as its smallest deviation? (put it in the first of the "pair_style deepmd“ line in the input file of lammps ).
In  lammps input file of model_devi step, 01 model is at the front by default. Does this mean that 01 model is most suitable for generating trajectories?
Looking forward to your reply, Thank you.

**Comment by njzjz** [comment, marked as answer]
> The rmse_f_trn of these four models

If you mean `rmse_f_trn ` in `lcurve.out`, that's just the RMSE for one batch of data and cannot be used to compare with other models.

To have a fair error, test the model on all data.

**Comment by h840473807** [comment]
Thank you for your reply, which solved my doubts.
